plugins/multilevel-linear-view/src/LinearGenomeMultilevelView/model.ts

```diff
@@ -0,0 +1,451 @@
+import { types, Instance } from 'mobx-state-tree'
+
+import {
+  clamp,
+  getSession,
+  parseLocString,
+  getContainingView,
+} from '@jbrowse/core/util'
+import { MenuItem } from '@jbrowse/core/ui'
+import { LinearGenomeViewStateModel } from '@jbrowse/plugin-linear-genome-view'
+import ExportSvgDlg from '@jbrowse/plugin-linear-genome-view/src/LinearGenomeView/components/ExportSvgDialog'
+import PluginManager from '@jbrowse/core/PluginManager'
+
+import { ElementId } from '@jbrowse/core/util/types/mst'
+import { TrackSelector as TrackSelectorIcon } from '@jbrowse/core/ui/Icons'
+import SyncAltIcon from '@mui/icons-material/SyncAlt'
+import VisibilityIcon from '@mui/icons-material/Visibility'
+import ZoomInIcon from '@mui/icons-material/ZoomIn'
+import LabelIcon from '@mui/icons-material/Label'
+import CloseIcon from '@mui/icons-material/Close'
+import AddIcon from '@mui/icons-material/Add'
+import VerticalAlignTopIcon from '@mui/icons-material/VerticalAlignTop'
+import VerticalAlignBottomIcon from '@mui/icons-material/VerticalAlignBottom'
+import PhotoCameraIcon from '@mui/icons-material/PhotoCamera'
+
+export interface BpOffset {
+  refName?: string
+  index: number
+  offset: number
+  start?: number
+  end?: number
+  coord?: number
+  reversed?: boolean
+  assemblyName?: string
+  oob?: boolean
+}
+
+export default function stateModelFactory(pluginManager: PluginManager) {
+  return types.compose(
+    'LinearGenomeMultilevelView',
+    pluginManager.getViewType('LinearGenomeView')
+      .stateModel as LinearGenomeViewStateModel,
+
+    types
+      .model({
```

Comment:
Instead of composing, you can build off the existing LGV model with `.state` and `.props`. That way TypeScript will know about the attributes on the model and you can remove a bunch of the `@ts-ignore`s in this (and maybe other) files. Something like this:

```suggestion
  return (
    pluginManager.getViewType('LinearGenomeView')
      .stateModel as LinearGenomeViewStateModel
  )
    .named('LinearGenomeMultilevelView')
    .props({
```